openmetadata-ui/src/main/resources/ui/src/utils/TimeUtils.ts

```diff
@@ -196,7 +196,10 @@ export const getPastDaysDateTimeMillis = (days: number) =>
 export const getFormattedDateFromSeconds = (
   timeStamp: number,
   format?: string
-) => DateTime.fromSeconds(timeStamp || 0).toFormat(format || 'dd/MMM HH:mm');
+) =>
+  DateTime.fromSeconds(timeStamp || 0)
+    .toUTC()
+    .toFormat(format || 'dd/MMM HH:mm');
```

Comment:
@ShaileshParmar11, Can you share some screenshots of where we are using this method?

Reply:
updated